src/App.js

```diff
@@ -1,13 +1,64 @@
-import React from 'react';
+import React from "react";
+import TodoList from "./components/TodoComponents/TodoList";
+import TodoForm from "./components/TodoComponents/TodoForm";
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      taskData: [
+        {
+          task: "Organize Garage",
+          id: Date.now(),
+          completed: false
+        },
+        {
+          task: "Bake Cookies",
+          id: Date.now(),
+          completed: false
+        }
+      ],
+      task: "",
+      id: "",
+      completed: ""
+    };
+  }
+
+  addTask = e => {
+    e.preventDefault();
+    const newTask = {
+      task: this.state.task,
+      id: Date.now(),
+      completed: this.state.completed
```

Comment:
```suggestion
      completed: false
```
They way you had it, this.state.completed is always an empty string, so all the new tasks you are adding are getting completed: ''. Instead, set each task to completed: false by default